docker-compose/monitor/README.md

```diff
@@ -321,3 +311,4 @@ $ curl http://localhost:16686/api/metrics/minstep | jq .
 [spanmetricsprocessor]: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/spanmetricsprocessor
 [spanmetricsconnector]: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/connector/spanmetricsconnector
 [processor-to-connector]: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/connector/spanmetricsconnector#span-to-metrics-processor-to-span-to-metrics-connector
```

Comment:
We could probably remove this too and assume everyone's switched over to the connector.

Reply:
Can I also remove the migration guide too @albertteoh ?

Reply:
Removed. Would you also like me to update the configuration of the spanmetrics connector replacing the migration guide there?